<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>898</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>1200</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 220);</string>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 167, 225)</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>20</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 36pt &quot;Arial&quot;;
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>ePAT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="settingButton1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>61</width>
       <height>10</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:3px;
font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="settingButton2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>61</width>
       <height>10</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:3px;
background-color: rgb(255, 255, 255);
font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="settingButton3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>61</width>
       <height>10</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:3px;
background-color: rgb(255, 255, 255);
font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>751</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
font: 16pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>280</y>
      <width>150</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Arial&quot;;
background-color: rgb(100, 167, 225);
border-radius:20px;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Search </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Please use the below search </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>340</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(100, 167, 225);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>340</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(100, 167, 225);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QComboBox" name="booleanBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(100, 167, 225);
border-radius:20px;</string>
    </property>
    <item>
     <property name="text">
      <string>OR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AND</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
